qoqo/src/lib.rs

```diff
@@ -107,7 +107,10 @@ pub static STRUQTURE_VERSION: OnceLock<String> = OnceLock::new();
 /// Struqture PlusMinusLindbladNoiseOperator object from local struqture package
 pub static STRUQTURE_OPERATOR: OnceLock<Py<PyAny>> = OnceLock::new();
 
-pub(crate) fn get_operator(py: Python, noise: &PlusMinusLindbladNoiseOperator) -> Py<PyAny> {
+pub(crate) fn get_operator<'py>(
+    py: Python<'py>,
+    noise: &PlusMinusLindbladNoiseOperator,
+) -> PyResult<Bound<'py, PlusMinusLindbladNoiseOperatorWrapper>> {
```

Comment:
I'm not sure this should work with a Wrapper as output - whether this wrapper is struqture 1 or 2 will depend on the installed version